package/package.json

```diff
@@ -81,9 +81,6 @@
     "react-native-url-polyfill": "^1.3.0",
     "stream-chat": "7.0.0"
   },
-  "peerDependencies": {
-    "react-native-svg": "^12.1.0"
-  },
```

Comment:
I think this will cause issues for our users. If they don't have this installed, it'll cause errors when we try to render SVGs

Reply:
Done